pkg/util/workqueue/default_rate_limiters.go

```diff
@@ -90,9 +90,18 @@ func (r *ItemExponentialFailureRateLimiter) When(item interface{}) time.Duration
 	r.failuresLock.Lock()
 	defer r.failuresLock.Unlock()
 
+	exp := r.failures[item]
+
+	// The maximum exponent maxExp is chosen such that the 'calculated' value never
+	// exceeds MaxInt64.
+	maxExp := int(math.Log10(float64(math.MaxInt64)/float64(r.baseDelay.Nanoseconds())) - 1)
+	if exp > maxExp {
```

Comment:
Hmm, maybe just cap it? the probelm is that time.Duration can't handle the float right
```
backoff := math.Pod10(r.failures[items])
if backoff >= math.MaxInt64 {
  return r.maxDelay
}
```